stock_picking_purchase_state/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Stock Picking Purchase State",
+    "summary": """
+        Add the Picking Status field to Purchase.
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "stock, purchase",
```

Comment:
Does it work? I mean using multiple categories for an addon description

Reply:
Yes it works, for example in a [stock-logistics-workflow ](https://github.com/OCA/stock-logistics-workflow/blob/acf795670aa5bfdcee32ba1ee526194f866a2951/stock_no_negative/__manifest__.py#L9)addon it has three categories